CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Enhancements
 * Updated to PHP CodeSniffer v3
 * Updated to Sami v4 for API documentation
 * Updated to PHPUnit v7
+* Replace abandonned Sami by Doctum
```

Comment:
Version 1.0.0 has already been released.

We need to add a new section to the ChangeLog for the changes since 1.0.0

Reply:
@williamdes can you add a `1.1.0` section with your addition?

@njh after we merged this, we could add further content to this section and release it. What do you think?

Reply:
Done :)